kotlin-react-query/src/main/kotlin/react/query/QueryErrorResetBoundary.kt

```diff
@@ -13,8 +13,8 @@ external interface QueryErrorResetBoundaryValue {
 
 external val useQueryErrorResetBoundary: () -> QueryErrorResetBoundaryValue
 
-external interface QueryErrorResetBoundaryProps : react.RProps {
-    var children: dynamic
+external interface QueryErrorResetBoundaryProps : react.Props {
```

Comment:
`PropsWithChildren`? I will fix it in generator

Reply:
https://react-query.tanstack.com/reference/QueryErrorResetBoundary
There should be render children, not `Array<ReactNode>`

Reply:
fixed anyway